### PR TITLE
update install with vagrant

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra/pull/544
https://github.com/open-orchestra/open-orchestra-docs/pull/56

### CODE REVIEW

File: en/developer_guide/install_with_vagrant.rst [hunk 1]
```diff
@@ -69,8 +69,9 @@ Install open-orchestra with composer:
 
 .. code-block:: bash
 
-    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
-    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/back/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/front/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-provision path/to/your/provisionning/folder -s dev
```
Comment (nicolasThal) [inline en/developer_guide/install_with_vagrant.rst]: As the repository open-orchestra-provision is not declared on packagist (it is not a php package), this command is not working

Comment (ngilain) [inline en/developer_guide/install_with_vagrant.rst]: So we have a problem as the provisionning stuff is no more on the open-orchestra bundle: this doc is a non-sense.

Or it should not be public and instead be placed in the open-orchestra-provision repo.

Comment (nicolasThal) [inline en/developer_guide/install_with_vagrant.rst]: No, this doc is not a non sense, the line just after this one explain how you can get the provisionning on your computer.

`composer` is only used to install php packages

Comment (ngilain) [inline en/developer_guide/install_with_vagrant.rst]: Oups sorry, didn't saw that. I update so.

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**